packages/website/docs/upgrading.md

```diff
@@ -21,37 +17,37 @@ import autocomplete from 'autocomplete.js';
 import { autocomplete } from '@algolia/autocomplete-js';
 ```
 
-## Params
+Read more about your [installation options](getting-started#installation) in the [guide on getting started](getting-started).
 
-Here's a list of params that will help you upgrade to from [Autocomplete v0](https://github.com/algolia/autocomplete.js/blob/45fa32d008620cf52bf4a90530be338543dfba7f/README.md#global-options) to v1:
+## Parameters
 
-- `autocomplete('#autocomplete', /* ... */)` → `autocomplete({ container: '#autocomplete', /* ... */ })`
-- `autoselect: true` → `defaultSelectedItemId: 0`
-- `autoselectOnBlur` is not needed anymore
-- `tabAutocomplete` is not supported because we implement the ARIA 1.1 form of the [combobox design pattern](https://www.w3.org/TR/wai-aria-practices/#combobox)
-- `hint` is not supported
-- `clearOnSelected` is now local to the [source](sources): `getItemInputValue: () => ''`
-- `dropdownMenuContainer` → `panelContainer`
-- `templates` (top-level) → `render` and `renderEmpty`
-- `cssClasses` → `classNames` where properties have changed
-- `keyboardShortcuts` is not supported as an option
-- `minLength` is replaced by only `openOnFocus` (which sets the minimum query length to either 0 or 1)
+These parameters and how you use them have changed from [Autocomplete v0](https://github.com/algolia/autocomplete.js/blob/45fa32d008620cf52bf4a90530be338543dfba7f/README.md#global-options):
 
-## Datasets
+| v0 | v1 |
+| ----------- | ----------- |
+| [`autocomplete('#autocomplete', /* ... */)`](https://github.com/algolia/autocomplete.js/blob/45fa32d008620cf52bf4a90530be338543dfba7f/README.md#global-options) | [`autocomplete({ container: '#autocomplete', /* ... */ })`](autocomplete-js/#container) |
+| [`autoselect: true`](https://github.com/algolia/autocomplete.js/blob/45fa32d008620cf52bf4a90530be338543dfba7f/README.md#global-options) | `defaultSelectedItemId: 0` |
```

Comment:
I couldn't find any API ref docs on `defaultSelectedItemId: 0`, I'm assuming there will be before launch?

Reply:
Oops, it's `defaultActiveItemId`.